### ISSUE TITLE
Update already uploaded repo

### ISSUE THREAD

**Issue by FireAlkazar** [CodeConnect/SourceBrowser#114]
I tried to explore http://sourcebrowser.io/Browse/dotnet/corefx, but it turned out that an old version is loaded, and the origin repository https://github.com/dotnet/corefx gone far away.
Re-uploading the repo did't do anything.
So, it would be nice to be able to update already uploaded repo.

**Comment by ddurschlag** [comment]
Agreed. As we look at the two versions of SourceBrowser (see issue #115), updates, especially incremental updates, are a big deal.
